src/v3/src/components/CaptchaContainer/CaptchaContainer.tsx

```diff
@@ -113,6 +114,9 @@ const CaptchaContainer: UISchemaElementComponent<{
   };
 
   if (captchaType === 'RECAPTCHA_V2') {
+    // Note: Unlike HCaptcha, we can't customize script URI with props.
+    // This can be done with global `recaptchaOptions` object:
+    // https://github.com/dozoisch/react-google-recaptcha#advanced-usage
```

Comment:
For Gen3 setting 
```
  recaptcha: {
    scriptSource: 'https://recaptcha.net/recaptcha/api.js',
  },
```
in widget options would have no effect.
Developer should use 
```
window.recaptchaOptions = {
      useRecaptchaNet: true,
};
```

Reply:
Should this be documented in the README ^ ?

Reply:
That's specific to SIW G3 because of a limitation of the React lib?